phylogenetic/defaults/all-lineages/config.yaml

```diff
@@ -0,0 +1,17 @@
+# Use 'Egypt 1951' as the reference and root, following Mencattelli et al, 2023
+# https://www.nature.com/articles/s41467-023-42185-7
+reference: "defaults/all-lineages/reference.gb"
+root: "mid_point"
+builds: ['all-lineages']
```

Comment:
Is there a reason why this is a list? Are there any configfiles / snakemake invocations which use multiple builds?

Reply:
There was some more discussion on this in https://github.com/nextstrain/WNV/pull/96#discussion_r2244239263. I see 2 approaches to reduce confusion around `builds` only ever being configured with a single entry:

1. Consolidate all 3 into a single multi-build config file and use `builds: [all-lineages, lineage-1A, lineage-2]`.
2. Make config files strictly single-build.

I agree it makes sense that all 3 have their own separate configs, so I went with (2).